tests/kamereon/test_kamereon_vehicle_action.py

```diff
@@ -32,3 +32,22 @@ def test_vehicle_action_response_attributes() -> None:
         "action": "start",
         "targetTemperature": 21.0,
     }
+
+
+def test_hvac_() -> None:
+    """Test vehicle action response attributes."""
+    response: models.KamereonVehicleHvacScheduleActionData = get_response_content(
+        f"{FIXTURE_PATH}/hvac-schedule.schedules.json",
+        schemas.KamereonVehicleHvacScheduleActionDataSchema,
+    )
+    print(response)
+    assert response.raw_data["data"]["attributes"]["schedules"][1]["activated"] is True
+    assert (
+        response.raw_data["data"]["attributes"]["schedules"][1]["monday"]["readyAtTime"]
+        == "T20:45Z"
+    )
+    assert response.raw_data["data"]["attributes"]["schedules"][2]["activated"] is False
+    assert list(response.raw_data["data"]["attributes"]["schedules"][2].keys()) == [
+        "id",
+        "activated",
+    ]
```

Comment:
You want to check the raw_data (just as simple dict compare) and then check the properties to ensure the schemas have parsed them correctly.